sleap/gui/commands.py

```diff
@@ -2119,12 +2119,23 @@ def do_action(cls, context: CommandContext, params: dict):
             "Generating list of suggested frames... " "This may take a few minutes.",
             context.app,
         )
+        if len(context.labels.videos) == 0:
+            print("Error: no videos to generate suggestions for")
+        else:
+            if (
+                params["target"] == "current video"
+            ):  # checks if current video is selected in gui
+                if context.state["video"] is not None:
+                    params["videos"] = [context.state["video"]]
+                else:
+                    params["videos"] = [context.labels.videos[0]]
```

Comment:
Can simplify with:
`params["videos"] = [context.labels.videos[0]] if context.state["video"] is None else [context.state["video"]]`